config/ssl/aws-net-ssl__dummy.adb

```diff
@@ -389,6 +389,14 @@ package body AWS.Net.SSL is
       raise Program_Error with Error_Message;
    end Set_Verify_Callback;
 
+   ----------------------------
+   -- Show_Session_Statistic --
+   ----------------------------
+
+   procedure Show_Session_Statistic
+     (Config   : SSL.Config;
+      Put_Line : not null access procedure (Line : String)) is null;
```

Comment:
This is not really Put_Line it is whatever the user decide it to be and it could clash (shadows) with Ada.Text_IO definitions. I would call the parameter `Report` to avoid confusion.

Reply:
Done